fix: Improved dynatrace integration guide (#634)
kyma-project/telemetry-manager: docs/user/integration/dynatrace/README.md

## Telemetry Module Setup

> Next, you set up the ingestion of custom span and Istio span data and optionally custom metrics based on OTLP.

Reviewer: ```suggestion
Next, you set up the ingestion of custom span and Istio span data, and, optionally, custom metrics based on OTLP.
```

### Ingest Traces

> To start ingesting custom pans and Istio spans, you need to enable the Istio tracing feature and then deploy a TracePipeline to enable the shipment to Dynatrace using the secrets defined above.

Reviewer: This sentence seems to be a summary of the steps that happen in this section? Then I'd shorten it a bit and add the information about the Secret closer to line 128 where it happens.

```suggestion
To start ingesting custom spans and Istio spans, you must enable the Istio tracing feature and then deploy a TracePipeline.
```

### Ingest Metrics (Experimental)

> To collect custom metrics, you usually use the Dynatrace annotation approach, as the Dynatrace OTLP integration is [limited](https://docs.dynatrace.com/docs/extend-dynatrace/opentelemetry/getting-started/metrics/ingest/migration-guide-otlp-exporter#migrate-collector-configuration). As long as your workload is conform to the limitations (not exporting histograms, using delta aggregation temporality), you can leverage the metric functionality to push OTLP metrics centralized to Dynatrace. Hereby, the prometheus feature of the MetricPipeline will not work as it will hit the limitations by design.

Reviewer: ```suggestion
To collect custom metrics, you usually use the Dynatrace annotation approach, because the Dynatrace OTLP integration is [limited](https://docs.dynatrace.com/docs/extend-dynatrace/opentelemetry/getting-started/metrics/ingest/migration-guide-otlp-exporter#migrate-collector-configuration). As long as your workload is conform to the limitations (not exporting histograms, using delta aggregation temporality), you can use the metric functionality to push OTLP metrics to Dynatrace. In this case, the Prometheus feature of the MetricPipeline does not work because it hits the limitations by design.
```

    EOF
    ```

> 1. Start pushing metrics to the metric gateway using delta aggregation temporality

Reviewer: ```suggestion
1. Start pushing metrics to the metric gateway using [delta aggregation temporality.](https://docs.dynatrace.com/docs/extend-dynatrace/opentelemetry/overview/metrics#temporality)
```
